staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_discovery.go

```diff
@@ -214,7 +232,7 @@ func (dm *discoveryManager) fetchFreshDiscoveryForService(gv metav1.GroupVersion
 		Path:              req.URL.Path,
 		IsResourceRequest: false,
 	}))
-	req.Header.Add("Accept", discovery.AcceptV2Beta1)
+	req.Header.Add("Accept", discovery.AcceptV2)
```

Comment:
Should both v2 and v2beta1 be accepted to tolerate skew?

Reply:
Fixed.